### PR TITLE
[Fix][e2e] Fix e2e failed due to server overload.

### LINKED ISSUE #15579: [Fix][e2e] Fix e2e failed due to server overload.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Fix e2e sometimes error

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by jackyyyyyssss**
## Purpose of the pull request
fix #15579
fix e2e sometimes error Reduce the probability of e2e errors. Currently, it has been found that some cases of e2e errors are as follows: 1. Because the automated testing browser cannot click on the switch, switch to a different writing method for clicking. 2. the resource is overloaded, increase the load parameter, first modify a few core methods to reduce the failure rate, and then modify based on the failed cases
image
image

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: dolphinscheduler-standalone-server/src/main/resources/application.yaml [hunk 1]
```diff
@@ -191,7 +191,7 @@ master:
   server-load-protection:
     enabled: true
     # Master max cpu usage, when the master's cpu usage is smaller then this value, master server can execute workflow.
-    max-cpu-usage-percentage-thresholds: 0.9
+    max-cpu-usage-percentage-thresholds: 1.5
```
Comment (ruanwenjun) [inline dolphinscheduler-standalone-server/src/main/resources/application.yaml:194]:
```suggestion
    max-cpu-usage-percentage-thresholds: 1.0
```
The max value of this parameter shouldld be 1.0.

File: dolphinscheduler-standalone-server/src/main/resources/application.yaml [hunk 2]
```diff
@@ -216,7 +216,7 @@ worker:
   server-load-protection:
     enabled: true
     # Worker max cpu usage, when the worker's cpu usage is smaller then this value, worker server can be dispatched tasks.
-    max-cpu-usage-percentage-thresholds: 0.9
+    max-cpu-usage-percentage-thresholds: 1.5
```
Comment (ruanwenjun) [inline dolphinscheduler-standalone-server/src/main/resources/application.yaml:219]:
```suggestion
    max-cpu-usage-percentage-thresholds: 1.0
```

**Comment by ruanwenjun** [comment]
Due to 3.2.1 has been released, and this will not affect the release package, I removed the 3.2.1 milestone.

File: dolphinscheduler-standalone-server/src/main/resources/application.yaml (hunk 2 above)
Comment (jackyyyyyssss) [reply dolphinscheduler-standalone-server/src/main/resources/application.yaml:219]: @ruanwenjun Hello  Sometimes the utilization rate of underlying resources can be greater than 1, so setting it to 1.5 can overload and cause e2e failure if set to 1
image

Comment (ruanwenjun) [reply dolphinscheduler-standalone-server/src/main/resources/application.yaml:219]: This should be a bug, we may need to use getTotalCpuUsedPercentage() to compare with maxCpuUsagePercentageThresholds at ServerLoadProtection

**Comment by ruanwenjun** [comment]
Since this PR is no longer updated, I submit #15803 to fix this problem, thanks for your PR.

**Pull request closed without merging**